meetings/2021-11-22.md

```diff
@@ -0,0 +1,93 @@
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* Nicolas AS
+* e Pavlik
+
+---
+
+## Agenda
+
+* [Open Pull Requests](https://github.com/solid/solid-oidc/pulls)
+  * [Add RS associated AS issuing Access Token PR#18](https://github.com/solid/solid-oidc/pull/18)
+* [Specify the client authentication method when using a Client Identifier](https://github.com/solid/authentication-panel/issues/201)
+
+---
+
+## Topics
+
+### Open Pull Requests
+
+https://github.com/solid/solid-oidc/pulls
+
+#### [Add RS associated AS issuing Access Token PR#18](https://github.com/solid/solid-oidc/pull/18)
+
+* eP: /Shares latest sequence diagram/. The changes include adding emojis to identify actors. The moment the client is authenticated is made explicit in the diagram too. The change also include removing the textual description of the auth code flow to point to the primer. Access token has bee replaced with ID token. Accordingly, the audience has been moved from solid (for Access token) to client ID (for ID token). The claim in the ID token associated to the client ID is proposed to be `azp`. The binding of the ID token to a DPoP key is also mentioned.
+* AC: I've look into the `azp` claim in the past. It is a good thing to have. I do not recall why we decided to use it. It seems like a good fit, so we should look more into it. It looks like it would desambiguate the claims we use.
+* eP: If we want to issue ID tokens scoped to a specific AuthZ server, then this information would be in the `aud` claim, and it would be good not to rely on this claim to discover the client ID.
+* eP: The PR does not include the discovery of the AuthZ server. Each actor presened in the diagram should be a conformance class so that they are individually described by the spec, in a slightly different structure. 
+* AC: +1 on reorganizing to frame the conformance classes.
+* eP: The restructuring could happen in a different PR, but in the current PR should include a description of the AuthZ server responsabilities, and examples of claims in the ID token. The fact that all of this is extensible by othe specs
+* AC: What approach were you thinking about for the authZ discovery ? including but not restricting to UMA as_uri ?
+* eP: Yes. I don't have other examples besides UMA, but there may be others.
+* AC: We want to be clear about expectations not to lock ourselves in a box. We should write something along the lines: the `as_uri` parameter must be included in the WWW-Authenticate header, unless the active authentication scheme requires a different discovery mechanism. 
+* eP: /Updates issue 25 accordingly/. Ideally we should merge this by the end of the month, but we can defer pain points in the existing issues.
+* AC: since this is a big change, we should make sure people are well-aligned on this. Should we reach out to CSS/NSS maintainers while this is still a draf, so that we have early feedback ?
```

Comment:
```suggestion
* AC: since this is a big change, we should make sure people are well-aligned on this. Should we reach out to CSS/NSS maintainers while this is still a draft, so that we have early feedback?
```